src/App.js

```diff
@@ -1,13 +1,207 @@
 import React, { Component } from 'react';
+import ContactList from './ContactList';
+import SearchBar from './SearchBar';
+import ShowContactForm from './ShowContactForm';
+import NetworkLoadAlert from './NetworkLoadAlert';
+import axios from 'axios';
 
 class App extends Component {
+  constructor() {
+    super();
+
+//  my initial state
+    this.state = {
+      searchText: '',
+      selectedIds: [],
+      contacts: []
+    };
+  }
+
+//  Sends GET request to retrieve initial state of contactlist. calls to loadingStart
+//  and loadingEnd functions to alert user of network loading event.
+  componentDidMount() {
+
+    this.loadingStart();
```

Comment:
This is looking pretty good!

One thing to notice here though, you are calling `this.loadingStart()` before your request, and `this.loadingEnd()` when the request is completed, nice working breaking those out into their own functions.

What would happen if an error occurred and the `.then` block never ran, only the `.catch` block?